src/seed_tools/utils/study_json_utils.ts

```diff
@@ -96,32 +97,20 @@ function jsonStudyReplacer(
     case 'end_date': {
       return new Date(value * 1000).toISOString();
     }
-    case 'channel':
-      if (options?.isChromium === true) {
-        return value;
-      }
-      return value.map((value: string): string => {
-        switch (value) {
-          case 'CANARY':
-            return 'NIGHTLY';
-          case 'STABLE':
-            return 'RELEASE';
-        }
-        return value;
-      });
-    case 'platform':
-      if (options?.isChromium === true) {
-        return value;
-      }
-      return value.map((value: string): string => {
-        return value.replace(/^PLATFORM_/, '');
-      });
+    case 'channel': {
+      return value.map((c: string) =>
+        channelToString(c, options?.isChromium === true),
+      );
+    }
+    case 'platform': {
+      return value.map((p: string) => platformToString(p));
```

Comment:
before this change the logic for revive and replace was in the same file close to each other. Now it's spread a bit. Can we do something here?

Reply:
https://github.com/brave/brave-variations/pull/1253/commits/5ef0b4e95c04499b7c0d4b55fdbaa9c382df5ae5